region/maridia/inner-green/East Sand Hall.json

```diff
@@ -999,10 +1129,55 @@
         "Gain run speed using between 19 and 33 tiles in the other room, and either roll in or do a controlled bounce to enter while descending close to the ground.",
         "Bounce near the end of the platform in front of the door, clearing the whole room at once.",
         "Samus will go over the Evirs, so there is no need for blue speed."
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "Cross-Room Tricky Spring Ball Bounce (Come in Running)",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 6.4375
+        }
+      },
+      "requires": [
+        {"notable": "Cross-Room Tricky Spring Ball Bounce (Right to Left)"},
+        "canTrickyDashJump",
+        "canTrickySpringBallBounce",
+        "canPlayInSand"
       ],
-      "devNote": [
-        "FIXME: This can work with more or less speed, but it probably requires bouncing on the pillars, doing a spring fling, and/or taking Evir damage.",
-        "Note that the blue spring ball bounce strat may generally be preferable since it can be done with less runway and tends to be more lenient."
+      "flashSuitChecked": true,
+      "note": [
+        "Gain max run speed with Speed Booster unequipped (extra run speed $2.0),",
+        "reequipping Speed Booster before the transition.",
+        "Jump over the first pillar and first Evir, bouncing on the second pillar;",
+        "this requires releasing jump when Samus is about 3 tiles past the first pillar.",
+        "From there, Samus should bounce over all the remaining Evirs."
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "Cross-Room Tricky Spring Ball Bounce (Come in With Spring Ball Bounce)",
+      "entranceCondition": {
+        "comeInWithSpringBallBounce": {
+          "speedBooster": true,
+          "movementType": "controlled",
+          "remoteAndLandingMinTiles": [[6.4375, 1]]
+        }
+      },
+      "requires": [
+        {"notable": "Cross-Room Tricky Spring Ball Bounce (Right to Left)"},
+        "canTrickyDashJump",
+        "canTrickySpringBallBounce",
+        "canPlayInSand"
```

Comment:
Highly recommend canSpringFling.  Getting out of the sand on the left is pretty hard.

Reply:
I couldn't make it to the end of the sand fall most of the time.  I'll have to look again

Reply:
Guess I was unmorphing too late.